### PR TITLE
 Bump rubygem-hammer_cli_katello to 0.15.0

### PR DESCRIPTION
**Pull request by akofink**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Follows https://github.com/theforeman/foreman-packaging/pull/2908

### CODE REVIEW

**Comment by ekohl** [comment]
If it's not released yet this shouldn't be merged. We are talking about how the nightlies should work, but we should never check in unreleased gems unless they're clearly marked as such.

**Comment by akofink** [comment]
How do I get this version hammer in nightly builds? Do I need to build it as 0.15.develop or something?

**Comment by ehelms** [comment]
@akofink Do you mind rebasing this?

**Comment by akofink** [comment]
Updated

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 %global gem_name hammer_cli_katello
 %global confdir hammer
-%global release 1
+%global release 0
```
Comment (ehelms) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: This can still be 1.

**Comment by ehelms** [comment]
Passing build with this change -- http://koji.katello.org/koji/taskinfo?taskID=129950
Merging as this is the last major piece of Rails 5.2 to be rebuilt.

**Pull request merged**